src/index.html

```diff
@@ -11,7 +11,156 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset>
+        <legend>Personal information</legend>
+
+        <!-- Surname -->
+        <div class="form-field">
+          <label for="Surname">Surname:</label>
+          <input type="text" name="Surname"
+          id="Surname" autocomplete="off"
+          >
+        </div>
+
+        <!-- Name -->
+        <div class="form-field">
+          <label for="Name">Name:</label>
+          <input type="text" name="Name"
+          id="Name" autocomplete="off"
```

Comment:
check out point 3 in the [checklist](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md)
```suggestion
          <input 
            type="text" 
            name="Name"
            id="Name" 
            autocomplete="off"
```